retis/src/core/probe/kernel/bpf/include/common.h

```diff
@@ -228,6 +296,9 @@ static __always_inline int chain(struct retis_context *ctx)
 
 	ctx->offsets = cfg->offsets;
 
+	if (extend_ctx(ctx))
+		return 0;
```

Comment:
I don't think ctx extensions should be able to make the whole probe fail. We could use an error message instead.